tests/utils/index.js

```diff
@@ -226,7 +226,7 @@ const saveDoc = async doc => {
   }
 };
 
-const saveDocs = async docs => {
+const saveDocs = async (docs, updateDocs = false) => {
```

Comment:
Minor rant: maybe this is one of those normal JS things that I have still not made peace with yet, but I see this pattern throughout our test code and it always makes me cringe.  

Using a parameter to toggle functionality that _fundamentally changes_ the behavior of a function seems like a code smell to me.  In this case, setting `updateDocs` will affect both the mutation of the `docs` param as well as changing what is actually returned from the function.  To me, this level of behavior change in a function just serves to add complexity and obscurity to its behavior and basically means that any consumers have to read/understand the entirety of the function to know how it is going to behave.  Then they have to keep this behavior in mind whenever they see the function being used....

To me, it seems better to avoid non-trivial differences in function behavior. If it cannot behave the same way all the time, then it probably should just be broken into multiple functions.  

---

In this case, I wonder if there is a reason we cannot just always update the `_rev` values on the provided docs?  If that is not possible then I think we are better off rolling back these changes and using separate function to apply the `_rev` values.

Reply:
Done, moved to another function.